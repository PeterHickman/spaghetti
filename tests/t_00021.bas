10 REM IF ... GOSUB test
20 for i = 1 to 10
30 if i < 5 gosub 60
40 next i
50 end
60 print i "is less than 5"
70 return
