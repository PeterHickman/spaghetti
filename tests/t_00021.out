1 is less than 5
2 is less than 5
3 is less than 5
4 is less than 5
